test/e2e/tests/app_test.go

```diff
@@ -75,7 +74,7 @@ func TestApp_Tx(t *testing.T) {
 		require.NoError(t, err)
 
 		hash := tx.Hash()
-		waitTime := 20 * time.Second
+		const waitTime = time.Minute
```

Comment:
I find this a bit worrying that someone could potentially have to wait for a minute for their tx to come through. However, if we want to go through with this it might also help to not query every second but perhaps every 5 seconds

Reply:
We we're previously at "wait 5s query every 1s" which seemed wrong too.

The node tests all run in parallel, (one test at a time per node,) and the tests (even if this one takes a minute) is a pretty small percentage of total run, but I think there's definitely a good move to be made at some point that we make the timeouts less across the e2e suite (and in general make things faster, but I think we should do that on a basis of everything passing,) 

I did change the transaction submission to be Synchronous, and it didn't fail in my last run of the full suite locally. I'm going to follow this up with adding tests of different transaction sumbissions at some point.